spec/support/ansible_shared/automation_manager/credential.rb

```diff
@@ -1,8 +1,8 @@
 require 'ansible_tower_client'
 
-shared_examples_for "ansible credential" do
+shared_examples_for "ansible credential" do |ansible_provider|
   let(:finished_task) { FactoryGirl.create(:miq_task, :state => "Finished") }
-  let(:manager)       { FactoryGirl.create(:provider_ansible_tower, :with_authentication).managers.first }
+  let(:manager)       { FactoryGirl.create(ansible_provider, :with_authentication).managers.first }
```

Comment:
So I tried it (defining outside), but it is [failing somehow](http://pastebin.test.redhat.com/468272). To succeed, I'll have to pass it `implicitly`.

I feel having the `shared_examples` parameters being explicitly `declared` in the block opening (as in `shared_examples_for do | ansible_provider | `) provides more clarity.